meteor/scripts/compute_difference_map.py

```diff
@@ -161,6 +161,12 @@ def main(command_line_arguments: list[str] | None = None) -> None:
         k_parameter_optimization=kparameter_metadata, tv_weight_optmization=tv_metadata
     )
 
+    if aggregate_metadata.tv_weight_optmization.optimal_negentropy <= 0.0:
+        log.warning(
+            "Final negentropy negative! Something has almost certainly gone wrong...",
```

Comment:
It's not clear to me that a negative negentropy value means anything other than the negentropy is close to zero and your estimator has some small numerical issues and bias. I would suggest softening the language considerably here. Something like:

"Negentropy estimate is negative. This indicates the map is very close to Gaussian and the true negentropy value is positive and close to zero."

Reply:
@kmdalton yep that's a good point. I guess "something has gone wrong" is ambiguous (bad). I was thinking: something has gone wrong in your experiment. But you are right that everything probably has gone right in the calculation! Let me improve it.

Reply:
Maybe something about there likely not being any statistically significant difference peaks?